<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="hLayoutGameMode">
     <item>
      <widget class="QLabel" name="lblGameMode">
       <property name="text">
        <string>Режим игры:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbGameMode">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Сложный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Стандартный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Простой</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Произвольный</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="hSpacerGameMode">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="hLayoutDatail">
     <item>
      <widget class="QGroupBox" name="gbDatail">
       <property name="title">
        <string>Детальные настройки</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_24">
        <item>
         <layout class="QHBoxLayout" name="hLayoutDatailSettings">
          <item>
           <layout class="QVBoxLayout" name="vLayoutLeft">
            <item>
             <widget class="QGroupBox" name="gbFieldSize">
              <property name="title">
               <string>Размеры поля</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_21">
               <item>
                <layout class="QVBoxLayout" name="vLayoutSize">
                 <item>
                  <layout class="QHBoxLayout" name="hLayoutWidth">
                   <item>
                    <widget class="QLabel" name="lblWidth">
                     <property name="text">
                      <string>Ширина:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="sbWidth">
                     <property name="minimum">
                      <number>5</number>
                     </property>
                     <property name="maximum">
                      <number>15</number>
                     </property>
                     <property name="value">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="hLayoutHeight">
                   <item>
                    <widget class="QLabel" name="lblHeight">
                     <property name="text">
                      <string>Высота:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="sbHeight">
                     <property name="minimum">
                      <number>5</number>
                     </property>
                     <property name="maximum">
                      <number>15</number>
                     </property>
                     <property name="value">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="vSpacerSize">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="gbPlayerInventory">
              <property name="title">
               <string>Инвентарь игрока</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_22">
               <item>
                <layout class="QVBoxLayout" name="vLayoutInventory">
                 <item>
                  <layout class="QHBoxLayout" name="hLayoutAmmunition">
                   <item>
                    <widget class="QLabel" name="lblAmmunition">
                     <property name="text">
                      <string>Патронов:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="sbAmmunition">
                     <property name="maximum">
                      <number>15</number>
                     </property>
                     <property name="value">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="hLayoutGrenade">
                   <item>
                    <widget class="QLabel" name="lblGrenade">
                     <property name="text">
                      <string>Гранат:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="sbGrenade">
                     <property name="maximum">
                      <number>10</number>
                     </property>
                     <property name="value">
                      <number>5</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="vSpacerInventory">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="gbRight">
            <property name="title">
             <string>Количество игровых объектов</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <layout class="QVBoxLayout" name="vLayoutObjects">
               <item>
                <layout class="QHBoxLayout" name="hLayoutRealTreasure">
                 <item>
                  <widget class="QLabel" name="lblRealTreasure">
                   <property name="text">
                    <string>Настоящих кладов:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sbRealTreasure">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="hLayoutFakeTreasure">
                 <item>
                  <widget class="QLabel" name="lblFakeTreasure">
                   <property name="text">
                    <string>Ложных кладов:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sbFakeTreasure">
                   <property name="maximum">
                    <number>2</number>
                   </property>
                   <property name="value">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="hLayoutArsenal">
                 <item>
                  <widget class="QLabel" name="lblArsenal">
                   <property name="text">
                    <string>Арсеналов:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sbArsenal">
                   <property name="maximum">
                    <number>3</number>
                   </property>
                   <property name="value">
                    <number>2</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="hLayoutHole">
                 <item>
                  <widget class="QLabel" name="lblHole">
                   <property name="text">
                    <string>Типов ям</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sbHole">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>2</number>
                   </property>
                   <property name="value">
                    <number>2</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="hLayoutExit">
                 <item>
                  <widget class="QLabel" name="lblExit">
                   <property name="text">
                    <string>Выходов:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sbExit">
                   <property name="minimum">
                    <number>4</number>
                   </property>
                   <property name="maximum">
                    <number>4</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="vSpacerObjects">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="hLayoutButtons">
     <item>
      <spacer name="hSpacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bAccept">
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
